man/systemd.netdev.xml

```diff
@@ -1975,6 +1978,107 @@
     </variablelist>
   </refsect1>
 
+  <refsect1>
+    <title>[BatmanAdvanced] Section Options</title>
+    <para>The [BatmanAdvanced] section only applies for
+    netdevs of kind <literal>batadv</literal> and accepts the
+    following keys:</para>
+
+    <variablelist class='network-directives'>
+      <varlistentry>
+        <term><varname>GatewayMode=</varname></term>
+        <listitem>
+          <para>The GatewayMode is a string of either <literal>off</literal>, <literal>server</literal> or
```

Comment:
Please use `<varname>` tag and append `=` after the seeting. I.e., `<varname>GatewayMode=</varname>`.
Or simply,
```
Takes one of <literal>off</literal>, <literal>server</literal>, or <literal>client</literal>.
```

Also, please add a comma before `or`.